.markdownlintignore

```diff
@@ -0,0 +1,3 @@
+.vale/**/*.md
+.github/**/*.md
```

Comment:
```suggestion
.vale
.github
```
doesn't markdownlint lint only markdown files? we might not actually need *.md here? we can just add both directories alone?

Reply:
yeah, it's sort of superfluous. Actually I notice there's no `.md` files under `.vale` now, and we should apply rules to `.github` directory as well.